src/components/KuadrantOverviewPage.tsx

```diff
@@ -141,6 +185,54 @@ const KuadrantOverviewPage: React.FC = () => {
     },
   ];
 
+  const gatewayTrafficColumns = [
+    {
+      title: t('plugin__kuadrant-console-plugin~Name'),
+      id: 'name',
+      sort: 'metadata.name',
+      transforms: [sortable],
+    },
+    {
+      title: t('plugin__kuadrant-console-plugin~Namespace'),
+      id: 'namespace',
+      sort: 'metadata.namespace',
+      transforms: [sortable],
+    },
+    {
+      title: t('plugin__kuadrant-console-plugin~Status'),
+      id: 'Status',
+    },
+    {
+      title: t('Total Requests'),
+      id: 'totalRequests',
+      sort: 'totalRequests',
+      transforms: [sortable],
+    },
+    {
+      title: t('Successful Requests'),
+      id: 'successfulRequests',
+      sort: 'successfulRequests',
+      transforms: [sortable],
+    },
+    {
+      title: t('Error Rate'),
+      id: 'errorRate',
+      sort: 'errorRate',
+      transforms: [sortable],
```

Comment:
think Rachel mentioned similar, but a default for these if null would be cool (table looks a bit weird if no errors)